agdk-cpal/src/lib.rs

```diff
@@ -147,7 +147,7 @@ fn android_main(app: AndroidApp) {
                         redraw_pending = false;
 
                         // Handle input
-                        app.input_events(|event| {
+                        app.input_events_iter().unwrap().next(|event| {
                             info!("Input Event: {event:?}");
                             InputStatus::Unhandled
                         });
```

Comment:
@rib this `next()` function returns a `bool`. Are we supposed to run it in a `while` loop, and if so can you put that in the doc-comment _and_ give this function `#[must_use]` to make it (much!) harder to overloop?